Frontend/src/Screens/GlobalComponents/NavBar/NavBar.css

```diff
@@ -0,0 +1,89 @@
+@media (max-width: 800px) {
+    .navbar-container {
+        width: 100%;
+        height: 50px;
```

Comment:
Please change any px to rem or %. You can use an online px to rem converter to get the desired measurements.